floss/render/default.py

```diff
@@ -145,23 +145,27 @@ def render_staticstrings(strings, ostream, verbose, disable_headers):
         unicode_offset_len = len(f"{unicode_strings[-1].offset}")
     offset_len = max(ascii_offset_len, unicode_offset_len)
 
-    render_sub_heading("FLOSS STATIC STRINGS: " + heading_style("ASCII"), len(ascii_strings), ostream, disable_headers)
-    for s in ascii_strings:
-        colored_string = string_style(s.string)
-        if verbose == Verbosity.DEFAULT:
-            ostream.writeln(colored_string)
-        else:
+    if verbose == Verbosity.DEFAULT:
+        render_sub_heading("FLOSS STATIC STRINGS: ASCII", len(ascii_strings), ostream, disable_headers)
+        for s in ascii_strings:
+            ostream.writeln(s.string)
+        ostream.writeln("")
+        render_sub_heading("FLOSS STATIC STRINGS: UTF-16LE", len(unicode_strings), ostream, disable_headers)
+        for s in unicode_strings:
+            ostream.writeln(s.string)
+    else:
+        render_sub_heading(
+            "FLOSS STATIC STRINGS: " + heading_style("ASCII"), len(ascii_strings), ostream, disable_headers
+        )
+        for s in ascii_strings:
+            colored_string = string_style(s.string)
             ostream.writeln(f"0x{s.offset:>0{offset_len}x} {colored_string}")
-    ostream.writeln("")
-
-    render_sub_heading(
-        "FLOSS STATIC STRINGS: " + heading_style("UTF-16LE"), len(unicode_strings), ostream, disable_headers
-    )
-    for s in unicode_strings:
-        colored_string = string_style(s.string)
-        if verbose == Verbosity.DEFAULT:
-            ostream.writeln(colored_string)
-        else:
+        ostream.writeln("")
+        render_sub_heading(
+            "FLOSS STATIC STRINGS: " + heading_style("UTF-16LE"), len(unicode_strings), ostream, disable_headers
+        )
+        for s in unicode_strings:
+            colored_string = string_style(s.string)
```

Comment:
this works, but there's quite a bit of duplicated code, can we simplify this before it gets out of hand (it's not easy to follow along what happens when here)